variants/Geekble_ESP32C3/pins_arduino.h

```diff
@@ -3,9 +3,13 @@
 
 #include <stdint.h>
 
-static const uint8_t LED_BUILTIN = 10;
-#define BUILTIN_LED  LED_BUILTIN // backward compatibility
-#define LED_BUILTIN LED_BUILTIN  // allow testing #ifdef LED_BUILTIN
+static const uint8_t    LED_BUILTIN = 10;
+#define BUILTIN_LED     LED_BUILTIN     // backward compatibility
+#define LED_BUILTIN     LED_BUILTIN     // allow testing #ifdef LED_BUILTIN
+
+static const uint8_t    SW_BUILTIN = 9;
+#define BUILTIN_SW      SW_BUILTIN      // backward compatibility
```

Comment:
Is there any sketch that uses `BUILTIN_SW` ? This is used for LED due to being like this in the past